src/steamship/client/client.py

```diff
@@ -106,6 +104,28 @@ def embed_and_search(
             space=space,
         )
 
+    def tag(
```

Comment:
I think this `tag` addition snuck in from not having rebased/merged against `main`?